### ISSUE TITLE
cluster-monitoring cannot list resource

### ISSUE THREAD

**Issue by josecelano** [rancher/rancher#20325]
Hi, I am trying to setup Cluster Monitoring Configuration. It seems the application it's deployed correctly but I can not see any data in Grafana.

The ` exporter-kube-state-cluster-monitoring` contains this logs:
```
E0520 08:15:32.218693 1 reflector.go:205] k8s.io/kube-state-metrics/pkg/collectors/builder.go:508: Failed to list *v1beta1.PodDisruptionBudget: poddisruptionbudgets.policy is forbidden: User "system:serviceaccount:cattle-prometheus:exporter-kube-state-cluster-monitoring" cannot list resource "poddisruptionbudgets" in API group "policy" at the cluster scope
```

The Grafana app gets a 502 is this AJAX request:
```
https://rancher.my-domain.com/k8s/clusters/c-t5dtb/api/v1/namespaces/cattle-prometheus/services/http:access-grafana:80/proxy/api/datasources/proxy/1/api/v1/query_range?query=sum%20by%20(instance)(rate(node_cpu_seconds_total%7Bmode%3D%27iowait%27%2Cinstance%3D%22undefined%22%7D%5B5m%5D))%20*%20100&start=1558340610&end=1558340940&step=30
```

---
| Useful | Info |
| :-- | :-- |
|Versions|Rancher `v2.2.3` UI: `v2.2.41` |
|Route|`workload`|

**Comment by coolersport** [comment]
This is how I fix it for the time being.

    kubectl edit clusterrole exporter-kube-state-cluster-monitoring -n cattle-prometheus
 
Add the following

    - apiGroups:
      - policy
      resources:
      - poddisruptionbudgets
      verbs:
      - list
      - get
      - watch

**Comment by loganhz** [comment]
fixed in monitoring 0.0.3.
